pocean/utils.py

```diff
@@ -305,14 +305,18 @@ def create_ncvar_from_series(ncd, var_name, dimensions, series, **kwargs):
     return v
 
 
+def _safe_date2num(dtime, units, calendar):
```

Comment:
This solution was inspired by what xarray does. Not sure why we never needed it here before :-/